content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -94,7 +94,7 @@ saitama_acc.withdraw(320)
 
 
 ## Inheritance
-It enables us to create a new class from an existing class. The new class is a specialized version of the existing class and it inherits all the non-private variables and methods of the existing class.
+It enables us to create a new class from an existing class. The new class (child class) is a specialized version of the existing class (parent class) and it inherits all the non-private variables and methods of the existing class.
```

Comment:
'Child classes can be created from parent classes, inheriting all the non-private variables and methods of their parents. They are specialized versions of the existing parent class.' ??

Reply:
I think the one above gives more details